CONTRIBUTING.md

```diff
@@ -16,20 +16,28 @@ With these steps you should be able to run Mars on your machine:
 
 1. Clone Mars: `git clone git@github.com:luksan47/mars.git`.
 2. Install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/). (Or other virtualization platforms supported by Vagrant. Don't forget to reconfigure the `provider` in the steps below if you do so.)
-3. Follow the instructions in the [First steps](https://laravel.com/docs/6.x/homestead#first-steps) section.
-4. Copy `.env.homesteadexample` to `mars/.env` and `Homestead.yaml.example` to `Homestead/Homestead.yaml`.
-. Modify the `Homestead.yaml` file by changing `/your/local/path/to/mars`.
+3. Follow the instructions in the [First steps](https://laravel.com/docs/8.x/homestead#first-steps) section:
+    * `vagrant box add laravel/homestead`
+    * `git clone https://github.com/laravel/homestead.git` from a folder where you want to set up Homestead
+    * go into this new directory
+    * `git checkout release`
+    * `init.bat` (`bash init.sh` on Linux)
+4. Set up Homestead: Copy `Homestead.yaml.example` from this repo to `homestead.yaml` in the Homestead directory. Modify this file by changing `folders: - map: /your/local/path/to/mars` .
+5. Set up Mars: Copy and rename `.env.example` to `.env`, and change these settings: 
```

Comment:
Would it be too much trouble to change the `Homestead.yaml.example` to work with the default settings?
Also, we should delete the `.env.homesteadexample`, since it's outdated.

Reply:
In `Homestead.yaml.example` only the local path (which is based on the system) is not set.